pkgs/by-name/Xr-Fragments/default.nix

```
@@ -0,0 +1,37 @@
+{
+  pkgs ? import <nixpkgs> { },
```

`pkgs` is already passed to the derivation, so there is no need to import it.

```suggestion
  pkgs
```

**PS:** importing it this way is bad for reproducibility since it will use the nixpkgs channel for whichever system is running this, instead of our nixpkgs, which might not be the same, and thus cause some undesirable effects.

That said, we want to explicitly pass each dependency instead of the whole `pkgs` anyways. For example:

```nix
{
  lib,
  stdenv,
  fetchgit,
  ...
}:
```